build.sh

```diff
@@ -13,6 +13,8 @@ else
     exit 1
 fi
 
+arch=$(uname -m)
```

Comment:
We should wrap this in single quotes, right? not sure why shellcheck isn't complaining.

Reply:
Assignments from `$()` like this don't need quotes. E.g.:

```
$ x=$(echo output with spaces)
$ echo $x
output with spaces
```